fairscale/nn/data_parallel/sharded_ddp.py

```diff
@@ -59,9 +58,6 @@ class ShardedDataParallel(nn.Module):
             (default: True) Check whether the parameters trainability (`requires_grad`) has changed and update both ShardedDDP
             and OSS automatically if this is the case. If set to False, `refresh_trainable()` needs to be called anytime
             a parameter is frozen or unfrozen.
-        reduce_fp16 (bool):
```

Comment:
this was broken and could be re-introduced later, not too complicated to do once the syncs are correct. I did try to keep this PR as small as possible